### DISCUSSION TITLE
PR should status checks

### DISCUSSION THREAD

**Discussion by devvspaces** [ageddesi/Mocked-API#197] [Ideas]
Should PR status checks be required? Currently if someone pushes a new PR and tests did not run successfully, you will still be able to merge. I think we should require status check to pass before merge can be complete. This will make sure that any new contributors won't be faced with failing tests when they want to contribute. 😉

**Comment by ageddesi** [comment]
We need to get the tests finally passing, as I want all tests to pass with each MR, this is something on my todo list :P

**Reply by devvspaces** [reply]
Yes, that's true, 

### Alas the last failing test:
![image](https://user-images.githubusercontent.com/77179231/195009569-b2b8b435-c913-49c5-8751-ea576f7b494e.png)

I don't know if anyone is working on it yet, I would check issues and PRs to see if anyone is working on it, else I will make a PR for resolving it

**Reply by ageddesi** [reply]
I do not think anyone is working on it atm from what I can see.

**Reply by devvspaces** [reply]
Yes, I see now, the product test is still failing as of today and it's the last failing test, I would debug it now and push it with PR #208 .

**Reply by devvspaces** [reply]
Then you should enforce the branch checks,  to make sure that any PR that fails tests is not merged.

**Reply by devvspaces** [reply]
I have done it, the product failing test is now passing. No new PR should be merged if they fail tests
